### ISSUE TITLE
Failing to enable WRITE_SECURE_SETTINGS permission

### ISSUE THREAD

**Issue by fnfontana** [bryan2894-playgrnd/SimpleWear#4]
Hello, I've downloaded the secondary app and when I try to do the adb procedure to enable the settings, it cannot execute the 'grant' command to enable the permissions.

My smartphone is the OnePlus Nord 2T 5G and my PC is Windows 10 21H2. I have used adb in the past with my old smartphone and I do understand it a bit. But I'm not figuring out what is going on here.

I have tried both the batch script that you have provided and manually copying and pasting the commands.

Here is the screenshot of what's happening:
![image](https://user-images.githubusercontent.com/8870142/190725573-b6a8aac6-b32e-42d8-9841-a8cdee27ee85.png)

**Comment by thewizrd** [comment]
Hello,

What version of Android are you using?

**Comment by fnfontana** [comment]
> Hello,
> 
> What version of Android are you using?

![Screenshot_2022-09-16-19-37-37-03_9492aa3750dca76abb7c25b39a5f1e8e.jpg](https://user-images.githubusercontent.com/8870142/190826755-2dedcba7-eca9-4f48-87d2-52043b193762.jpg)

**Comment by Adam4CZ** [comment]
Hello @thewizrd 

today I installed SimpleWear app and the add-on for settings and I ran into the same problem.
I have the same phone (OnePlus Nord 2T 5G) with same OS version (Oxygen OS 12.1 on Android 12).
Is there any update or solution for this problem?
Thank you.

![Screenshot 2023-03-06 102346](https://user-images.githubusercontent.com/15090021/223069517-eb883d67-fa94-468b-a71a-703a8e3e8380.png)
![Screenshot 2023-03-06 102445](https://user-images.githubusercontent.com/15090021/223069564-cfe89bcf-4865-4b2c-af46-0040c45ea9c5.png)
![Screenshot_2023-03-06-10-22-17-12_9492aa3750dca76abb7c25b39a5f1e8e](https://user-images.githubusercontent.com/15090021/223069576-6057ee5e-748a-4c40-87f0-577188aa6cbc.jpg)
